src/components/Search/Search.scss

```diff
@@ -0,0 +1,20 @@
+.search {
+  @include mdc-text-field-label-color(rgba(128, 128, 128, 0.719));
+  @include mdc-text-field-fill-color(white);
+  padding-right: -5rem;
+  border-radius: 2rem;
+  height: 4rem;
+  width: 20rem;
+  position: absolute;
```

Comment:
component itself should not know about its positioning - please move positioning rules to `main__search `